docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -0,0 +1,58 @@
+# Creating a new network from the Cleura Cloud Control Panel
+
+Before creating a server in {{extra.brand}} Cloud, you need to have a 
+network to make the new server a member of. You may have more than one 
+network per region, so let us now walk through creating a new network 
+using the {{extra.brand}} Cloud Control Panel. Fire up your favorite web 
+browser, navigate to the [Cleura Cloud](https://cleura.cloud) page, and 
+login into your {{extra.brand}} account.
+
+On the top right-hand side of the {{extra.brand}} Cloud Control Panel, 
+click the _Create_ button. A new pane will slide into from the 
+right-hand side of the browser window, titled ---unsurprisingly 
```

Comment:
Please remove the "unsurprisingly enough" bit. In general, we want to avoid phrases that sound like something is obvious or excessively easy, because obviousness and ease are quite subjective — and such phrases might come across as condescending to people who find something not so obvious and easy.

Reply:
Sure, removed it. That was a joke, actually, one that I was pretty sure would not survive the final version of the how-to. But I included it anyways :)